backend/requirements/dev-requirements.in

```
@@ -4,3 +4,4 @@ bandit
 black
 flake8
 model_bakery
```

Fix: I think we're missing `mypy` here

```suggestion
model_bakery
mypy
```